runtime/float.go

```diff
@@ -260,7 +261,8 @@ func floatRDivMod(f *Frame, v, w *Object) (*Object, *BaseException) {
 }
 
 func floatRepr(f *Frame, o *Object) (*Object, *BaseException) {
-	return NewStr(strconv.FormatFloat(toFloatUnsafe(o).Value(), 'g', -1, 64)).ToObject(), nil
+	v := unsignPositiveInf(strings.ToLower(strconv.FormatFloat(toFloatUnsafe(o).Value(), 'f', -1, 64)))
+	return NewStr(v).ToObject(), nil
```

Comment:
The question is, why u'1e+06' is falling back in float.__repr__? I should check unicode implementation